README.md

```diff
@@ -50,14 +50,21 @@ docker build -t lidofinance/oracle .
+|----------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|----------|-------------------------|
+| `EXECUTION_CLIENT_URI`                 | URI of the Execution Layer client                                                                                                                        | True     | `http://localhost:8545` |
+| `CONSENSUS_CLIENT_URI`                 | URI of the Consensus Layer client                                                                                                                        | True     | `http://localhost:5052` |
+| `KEYS_API_URI`                         | URI of the Keys API                                                                                                                                      | True     | `http://localhost:8080` |
+| `LIDO_LOCATOR_ADDRESS`                 | Address of the Lido contract                                                                                                                             | True     | `0x1...`                |
+| `ALLOW_NEGATIVE_REBASE_REPORTING`      | If 'False', a report with negative cl rebase would not be reported                                                                                       | False    | `True`                  |
+| `MEMBER_PRIV_KEY`                      | Private key of the Oracle member account                                                                                                                 | False    | `0x1...`                |
+| `FINALIZATION_BATCH_MAX_REQUEST_COUNT` | The size of the batch to be finalized per request (The larger the batch size, the more memory of the contract is used but the fewer requests are needed) | False    | `1000`                  |
+| `ALLOW_NEGATIVE_REBASE_REPORTING`      | Allow the oracle to report negative rebase                                                                                                               | False    | `True`                  |
+| `TX_GAS_MULTIPLIER`                    | A gas multiplier is used to ensure that there is enough extra gas to execute the transaction.                                                            | False    | `1.75`                  |
+| `CYCLE_SLEEP_IN_SECONDS`               | The time between cycles of the oracle's activity                                                                                                         | False    | `12`                    |
+| `SUBMIT_DATA_DELAY_IN_SLOTS`           | The delay between the latest blockstamp and when the oracle submits the report                                                                           | False    | `6`                     |
```

Comment:
The difference in slots between submit data transactions from Oracles. Used to prevent simultaneous sending of transactions and, as a result, transaction reverts.